docs/source/installation/troubleshooting.rst

```diff
@@ -12,3 +12,62 @@ uses. Which can be done by running:
 .. code-block:: bash
 
   fmtutil -sys --all
+
+
+Installation does not support converting PDF to SVG?
+----------------------------------------------------
+
+First, make sure your ``dvisvgm`` version is at least 2.4:
+
+.. code-block:: bash
+
+  dvisvgm --version
+
+
+If you do not know how to update ``dvisvgm``, please refer to your operating system's documentation.
+
+Second, check whether your ``dvisvgm`` supports PostScript specials. This is needed in order to convert from PDF to SVG.
+
+.. code-block:: bash
+
+  dvisvgm -l
+
+
+If the output to this command does **not** contain ``ps  dvips PostScript specials``, this is a bad sign.
+In this case, run
+
+.. code-block:: bash
+
+  dvisvgm -h
+
+
+If the output does **not** contain ``--libgs=filename``, this means your ``dvisvgm`` does not currently support PostScript. You must get another binary.
+
+If, however, ``--libgs=filename`` appears in the help, that means that your ``dvisvgm`` needs the Ghostscript library in order to support PostScript. Search for ``libgs.so`` (on Linux, probably in ``/usr/local/lib`` or ``/usr/lib``) or ``gsdll32.dll`` (on 32-bit Windows, probably in ``C:\windows\system32``) or ``gsdll64.dll`` (on 64-bit Windows, probably in ``c:\windows\system32`` -- yes 32) or ``libgsl.dylib`` (on Mac OS, probably in ``/usr/local/lib`` or ``/opt/local/lib``). Please look carefully, as the file might be located elsewhere.
+
+As soon as you have found the library, try (on Mac OS or Linux)
+
+.. code-block:: bash
+
+  export LIBS=<path to your library including the file name>
+  dvisvgm -l
+
+or (on Windows)  
+
+.. code-block:: bash
+
+  set LIBS=<path to your library including the file name>
```

Comment:
This creates a temp variable for the terminal session. Is it indented?

Reply:
Yes. It allows the user to test whether this solves the problem. If it does, they should check their documentation on how to persistently set this. I think it would be `setx` in recent versions of Windows. However, `setx` would not affect shells that are currently open. (And one would not want to persistently set a variable before knowing that this actually has the desired effect.)

However, if you have a better suggestion, you are more than welcome to add it. I only rarely use Windows and have never used manim there.